cmake/kconfig.cmake

```diff
@@ -89,6 +89,7 @@ set(
   ${BOARD_DEFCONFIG}
   ${CONF_FILE_AS_LIST}
   ${OVERLAY_CONFIG_AS_LIST}
+  ${shield_conf_files}
```

Comment:
I'd think shield_conf_files were at a similar priority to BOARD_DEFCONFIG, so after BOARD_DEFCONFIG.

Reply:
Yes, you are correct.  Fixed.